implementations/scripts/issue_github.py

```diff
@@ -0,0 +1,125 @@
+from copy import deepcopy
+
+from implementations.scripts.metric import Metric
+from implementations.scripts.utils import str_to_date
+
+
+class IssueGithub(Metric):
+    """
+    Generic class for issue metrics without pandas structure
+    """
+
+    def __init__(self, items, date_range=(None, None), reopen_as_new=False):
+        """
+        Constructor of this class
+
+        :param items: list of issues fetched by Perceval into a flat dictionary
+        :param date_range: Tuple, containing a range of date
+        :param reopen_as_new: allow to consider that reopen an issue is a new issue
+        """
+        self.since, self.until = date_range
+        super().__init__(items)
+
+        if reopen_as_new:
+            self.items = self._update_with_reopened_items(self.items)
```

Comment:
Maybe you could set `reopen_as_new` to `False`, and add a comment based on our discussion on why reopen_as_new doesn't work? WDYT?

The doc string of the function can have the extended note for consistency. Let me know if this is not clear.

Reply:
The `reopen_as_new` parameter is already set to False by default. Okay, I'll add this comment into the doc string.

Reply:
@xavierbol, I wanted to permanently set this to False, though now, I think it's better it remains this way, now that we have added the note in the doc-string.